mandatory/4-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+  return price*1.2;
```

Comment:
As Alun's session tonight, maybe need to set a variable here which is more clear:
  let priceWithTax = price * 1.2
  return priceWithTax;